### PR TITLE
change macro CONFIG_BINFMT_DISABLE to CONFIG_BINFTM_ENABLE

### PR DESCRIPTION
**Pull request by drashti304**
Due to CONFIG_BINFMT_DISABLE macro, binfmt is enabled by default even though
it is not needed.

This patch changes CONFIG_BINFMT_DISABLE to CONFIG_BINFTM_ENABLE
to make it disable by default.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [12ee84de9bf47d63dd3517e6926e108da9759ea0] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [12ee84de9bf47d63dd3517e6926e108da9759ea0] - Code Rule Check OK.

File: build/configs/imxrt1020-evk/tc_protected/defconfig [hunk 1]
```diff
@@ -899,7 +899,8 @@ CONFIG_C99_BOOL8=y
 #
 # Binary Loader
 #
-# CONFIG_BINFMT_DISABLE is not set
+
```
Comment (sunghan-chang) [inline build/configs/imxrt1020-evk/tc_protected/defconfig]: Please remove this empty line

**Comment by seinfra** [comment]
Target : [f6a76047c9ce872e5c4d9dd795356a3779b8e90b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f6a76047c9ce872e5c4d9dd795356a3779b8e90b] - Code Rule Check OK.

File: build/configs/imxrt1020-evk/tc_protected/defconfig (hunk 1 above)
Comment (drashti304) [reply build/configs/imxrt1020-evk/tc_protected/defconfig]: done

**Pull request merged**